apps/glusterfs/node_entry.go

```diff
@@ -159,13 +159,16 @@ func (n *NodeEntry) IsDeleteOk() bool {
 	return true
 }
 
+func (n *NodeEntry) ErrConflictFmt() error {
```

Comment:
Interesting style, but I would name it something different because ErrXXX is normally for errors and XXXFmt() is normally for something like the string in Printf( "fmt..").  How about:

```go
func (n *XXX) ErrorMsg( err error ) string {

   switch err {
   case ErrConflict:
       return fmt.Sprintf("...")
...
   }
}
```